scripts/helpers.js

```diff
@@ -0,0 +1,75 @@
+// Not intended to be run as a script
+
+var shell = require('shelljs');
+
+var assignments = [];
+var exerciseDirs = shell.ls('-d', 'exercises/*');
+for(i = 0; i < exerciseDirs.length; i++) {
+  assignments.push(exerciseDirs[i].split('/')[1]);
+}
+
+function prepare(assignment) {
+  if(!assignment) {
+    shell.echo('assignment not provided');
+    shell.exit(1);
+  }
+  var exampleFile = 'exercises/' + assignment + '/example.js';
+  var specFile = 'exercises/' + assignment + '/' + assignment + '.spec.js'
+
+  var inDir = 'exercises/' + assignment;
+  var outDir = 'tmp_exercises/' + assignment;
+
+  shell.mkdir('-p', 'tmp_exercises/lib');
+  shell.cp(exampleFile, 'tmp_exercises/' + assignment + '.js')
```

Comment:
Same as before regarding path joining; in this case I probably would use a string template literal